fcd/main.cpp

```diff
@@ -80,6 +80,8 @@ DEFINE_string(frameworks, "",
               "Apple framework dirs to be used for declarations");
 DEFINE_string(includes, "", "Directories to search headers in");
 
+DECLARE_string(callconv);
```

Comment:
Maybe make this `calling_convention`, or maybe `default_calling_convention`. I like command-line arguments to be, as much as possible, self-describing, kind of like variable names.

Reply:
And lol about me using `arch` and `os`.. which are terse. Humans are inconsistent!

Reply:
This is actually another leftover from my experiments with the original fcd code. Deleted in 
7effdc6. But comment noted for future.